### DISCUSSION TITLE
Picture is Turning/rotating with the Time

### DISCUSSION THREAD

**Discussion by maxis1601** [jomjol/AI-on-the-edge-device#2257] [General]
### The Problem

The image in the Overview is turning around, without a turn of the Camera, and the marks are stable. (like the first screenshot)
Then i get this Error: "Rate too high - Read: 7121.44 - Pre: 162.39 - Rate: 6959.05" because it read false numbers.
If i look in my settings, all is fine. Anyone have an idea?

### Version

v15.1.1

### Logfile

```shell
[2d03h48m59s] 2023-03-27T16:27:46	<ERR>	[POSTPROC] main: Raw: 2120.45, Value: , Status: Rate too high - Read: 2120.45 - Pre: 162.39 - Rate: 1958.06
[2d03h50m58s] 2023-03-27T16:29:45	<ERR>	[POSTPROC] main: Raw: N120.41, Value: , Status: Neg. Rate - Read:  - Raw: N120.41 - Pre: 162.39
```

### Screenshots

![grafik](https://user-images.githubusercontent.com/26626038/227973929-ce551bc9-5701-49e9-b2c2-89da6a7508a2.png)
 Picture how it is in Main Screen after a while.

![grafik](https://user-images.githubusercontent.com/26626038/227974091-aa6c5e10-eef3-4bc9-8166-59d8df3c7806.png) Reference Image

![grafik](https://user-images.githubusercontent.com/26626038/227974026-b0d2319f-ff90-4b55-9729-c6c1ea93479b.png)
 Alignment Mark 1
[
![grafik](https://user-images.githubusercontent.com/26626038/227973812-93f4fc72-c8d0-4a22-a3ce-ec30a1b8d3d3.png)
](url)
 Alignment Mark 2

![grafik](https://user-images.githubusercontent.com/26626038/227974154-41d05398-3646-44a4-9287-ac2d77d3884e.png)
Digit ROI's

![grafik](https://user-images.githubusercontent.com/26626038/227974228-7ebe069c-07c8-4e43-8e24-176124ed92bb.png)
Analog ROI's

**Comment by ghost** [comment]
Do not use unsharpened regions for alignment. Instead of MOM I would use the M21. Try to use add contrast in the configuration. (+1 or +2)

**Comment by caco3** [comment]
and put the marks farher apart
